ui/store/actions.js

```diff
@@ -1398,6 +1398,23 @@ export function addToken(
   };
 }
 
+export function updateTokenType(tokenAddress) {
+  return (dispatch) => {
+    dispatch(showLoadingIndication());
+    return new Promise((resolve, reject) => {
+      background.updateTokenType(tokenAddress, (err, tokens) => {
```

Comment:
It would be better to use the `promisifiedBackground` for all new actions - that's the one we're switching everything to use.